_board/bradanlanestudio_coin_m0.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: download
+board_id: "bradanlanestudio_coin_m0"
+title: "Bradán Lane STUDIO Coin M0 Download"
+name: "Bradán Lane STUDIO Coin M0"
+manufacturer: "Bradán Lane STUDIO"
+board_url: https://aosc.cc/eccn2025
+board_image: "bradanlanestudio_coin_m0.jpg"
+date_added: 2024-11-22
+family: atmel-samd
+features:
```

Comment:
`USB-C` can be added here as a feature, that will make the filter on the downloads page find it.